code/game/objects/effects/spawners/random_spawners.dm

```diff
@@ -300,10 +300,9 @@
 		/obj/structure/falsewall/reinforced = 1)
 
 /obj/effect/spawner/random_spawners/syndicate/layout/door/vault
-	name = "60pc vaultdoor 20pc falsewall 20pc wall"
-	result = list(/obj/machinery/door/airlock/hatch/syndicate/vault = 6,
-		/turf/simulated/wall/r_wall = 2,
-		/obj/structure/falsewall/reinforced = 2)
+	name = "80pc vaultdoor 20pc wall"
```

Comment:
Eliminating the potential false walls into armory addresses 3 things at once. First, it stops the shield looking wonky, because shields appeared over normal walls, but under false walls, making the false walls obvious, and the shield look buggy. Second, it makes the sentry bot more reliably able to path through the depot, which it did not always do before (sentry bot cannot open falsewalls). Third, it discourages players from opening up the armory while the shield is still up (an action that will disadvantage them, and which false walls were improperly encouraging).